Extension/src/LanguageServer/protocolFilter.ts

```diff
@@ -14,18 +14,18 @@ import { onDidChangeActiveTextEditor, processDelayedDidOpen } from './extension'
 
 export function createProtocolFilter(clients: ClientCollection): Middleware {
     // Disabling lint for invoke handlers
-    let defaultHandler: (data: any, callback: (data: any) => void) => void = (data, callback: (data) => void) => { clients.ActiveClient.notifyWhenReady(() => callback(data)); };
+    let defaultHandler: (data: any, callback: (data: any) => void) => void = (data, callback: (data: any) => void) => { clients.ActiveClient.notifyWhenReady(() => callback(data)); };
     /* tslint:disable */
```

Comment:
Is this still needed?

Reply:
Is what still needed?  defaultHandler?  It is still referenced before.